doc/content/guides/connecting-gateways/multitechconduit/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "MultiTech Conduit AEP"
+description: ""
+weight: 8
+---
+
+The [MultiTech MultiConnect® Conduit™](http://www.multitech.net/developer/products/multiconnect-Conduit-platform/) is a configurable, scalable cellular communications gateway for industrial IoT applications. The technical specifications of this Conduit can be found in the software guide in the [official documentation](http://www.multitech.net/developer/products/multiconnect-conduit-platform/conduit/) page.
+
+This guide will help you set up the gateway to communicate over {{%tts%}}.
+<!--more-->
+{{< figure src="001_Multitech_Conduit_AEP.png" alt="MultiTech MultiConnect Conduit AEP Gateway" >}}
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. Multitech Conduit AEP Gateway connected to internet and running with [latest firmware](http://www.multitech.net/developer/downloads/).
+
+> This article addresses the minimum configuration required to connect your Multitech Conduit AEP model to The Things Stack.
+> For the first-time hardware and software setup, you can refer to the appropriate configuration guide on the [Multitech documentation](http://www.multitech.net/developer/products/multiconnect-conduit-platform/conduit/) page.
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}).
+
+> **Note:** The **EUI** of the gateway can be found at the bottom of the gateway under the field **NODE LORA**.
```

Comment:
Please remove the use of **strong**, for consistency on the website we only use it while referring to buttons.